### PR TITLE
Simplify tests

### PR DESCRIPTION
**Pull request by ngaut**

### CODE REVIEW

**Comment by ngaut** [comment]
@morefreeze PTAL

**Comment by ngaut** [comment]
@qiuyesuifeng PTAL

File: ticlient_test.go [hunk 1]
```diff
@@ -26,6 +27,10 @@ var (
 	clusterID    = flag.Int("cluster", 1, "cluster ID")
 )
 
+func enc_to_key(s string) []byte {
```
Comment (siddontang) [inline ticlient_test.go]: the function is not go style.

**Comment by morefreeze** [comment]
LGTM

**Pull request merged**